pyproject.toml

```diff
@@ -6,22 +6,51 @@
 # verbose = true
 line-length = 80
 skip-string-normalization = true
+
+[tool.darker]
+src = [
+    "kobo",
+    "hub",
+    "kpi",
+]
+isort = true
+
 [tool.isort]
 profile = "black"
 line_length = 80 # same as black
 known_first_party = "kobo"
 no_lines_before = ["LOCALFOLDER"]
 
 [tool.ruff]
-line-length = 80
+line-length = 88
```

Comment:
If we're capping this at 88 we should cap black and isort at 88 as well, or put a comment explaining why they are different.

Reply:
I know we discussed this on an earlier PR, but I still don't understand why we kept our black config at 80

Reply:
We want the formatter to break at 80 but we don't want the linter to complain if we manually keep in purpose at line between 80 and 90 (as explained before) without adding `#noqa` or even worse `#fmt off/on`.

Reply:
We should discuss this more at a later date. I think we should add `#fmt off` anywhere we are purposefully ignoring formatting rules

Reply:
To put it another way: I think we need to decide firmly whether 88 is an acceptable limit. If it is, it should be the limit in all cases and we don't need to warn at 80. If not, we should standardize the line-length at 80 and explicitly call out lines that break that rule with `#fmt off` and an explanation

Reply:
Let's not waste our time on this. Let's use 88 everywhere since that's the default for Black and ruff. Moreover, I really don't want to add `#fmt on/off` each time my line is just 2 characters above 80.